imagetest/test_suites/image_boot/image_boot_test.go

```diff
@@ -54,6 +63,14 @@ func TestGuestRebootOnHost(t *testing.T) {
 }
 
 func TestGuestSecureBoot(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		testWindowsGuestSecureBoot(t)
```

Comment:
as a rule we shouldn't pass `t` around, but use the usual error paradigm and have error handling in this function

Reply:
yeah if you want to keep the functions split up as written, i suggest they don't take any args but return error, then this function simply uses the `if err != nil` paradigm and all the use of the `t` object remains in this function